python/python/lancedb/remote/table.py

```diff
@@ -354,19 +354,19 @@ def _do_merge(
         params["on"] = merge._on[0]
         params["when_matched_update_all"] = str(merge._when_matched_update_all).lower()
         if merge._when_matched_update_all_condition is not None:
-            params[
-                "when_matched_update_all_filt"
-            ] = merge._when_matched_update_all_condition
+            params["when_matched_update_all_filt"] = (
+                merge._when_matched_update_all_condition
+            )
         params["when_not_matched_insert_all"] = str(
             merge._when_not_matched_insert_all
         ).lower()
         params["when_not_matched_by_source_delete"] = str(
             merge._when_not_matched_by_source_delete
         ).lower()
         if merge._when_not_matched_by_source_condition is not None:
-            params[
-                "when_not_matched_by_source_delete_filt"
-            ] = merge._when_not_matched_by_source_condition
+            params["when_not_matched_by_source_delete_filt"] = (
+                merge._when_not_matched_by_source_condition
+            )
```

Comment:
Can you revert the changes to this file?  They are breaking our lint job (maybe you are using a different version of `ruff`?)

If it becomes trouble then let me know and I will fix after the fact.

Reply:
I am using ruff 0.4.6,  and I am afraid the lint job does not use the latest ruff. I will revert those changes.

BTW, if your lint job is using a specific version of ruff, I think it would be better to specify it in README file.